### PR TITLE
Implement Object.prototype.{__lookupGetter__, __lookupSetter__}

### PR DESCRIPTION
**Pull request by szilagyiadam**
__lookupGetter__ is based on ECMA-262 v11, B.2.2.4
__lookupSetter__ is based on ECMA-262 v11, B.2.2.5

JerryScript-DCO-1.0-Signed-off-by: Adam Szilagyi aszilagy@inf.u-szeged.hu

### CODE REVIEW

File: jerry-core/ecma/builtin-objects/ecma-builtin-object-prototype.c [hunk 1]
```diff
@@ -203,6 +205,94 @@ ecma_builtin_object_prototype_object_property_is_enumerable (ecma_object_t *obj_
   return ecma_make_boolean_value (is_enumerable);
 } /* ecma_builtin_object_prototype_object_property_is_enumerable */
 
+#if ENABLED (JERRY_ESNEXT)
+/**
+ * The Object.prototype object's '__lookupGetter__' and '__lookupSetter__' routine
```
Comment (galpeter) [inline jerry-core/ecma/builtin-objects/ecma-builtin-object-prototype.c:210]: I think we should add the ANNEXB guard for these features.

File: jerry-core/ecma/builtin-objects/ecma-builtin-object-prototype.c [hunk 2]
```diff
@@ -307,6 +397,16 @@ ecma_builtin_object_prototype_dispatch_routine (uint16_t builtin_routine_id, /**
   {
     ret_value = ecma_builtin_object_prototype_object_has_own_property (obj_p, prop_name_p);
   }
+#if ENABLED (JERRY_ESNEXT)
```
Comment (rerobika) [inline jerry-core/ecma/builtin-objects/ecma-builtin-object-prototype.c:400]: Missing annexb guard

File: jerry-core/ecma/builtin-objects/ecma-builtin-object-prototype.c [hunk 3]
```diff
@@ -51,7 +51,9 @@ enum
   ECMA_OBJECT_PROTOTYPE_IS_PROTOTYPE_OF,
   ECMA_OBJECT_PROTOTYPE_HAS_OWN_PROPERTY,
   ECMA_OBJECT_PROTOTYPE_PROPERTY_IS_ENUMERABLE,
-  ECMA_OBJECT_PROTOTYPE_SET_PROTO
+  ECMA_OBJECT_PROTOTYPE_SET_PROTO,
+  ECMA_OBJECT_PROTOTYPE_LOOKUP_GETTER,
```
Comment (rerobika) [inline jerry-core/ecma/builtin-objects/ecma-builtin-object-prototype.c:55]: Ditto.

**Comment by dbatyai** [comment]
Please rebase.

**Pull request merged**